pallets/dapp-staking-v3/src/lib.rs

```diff
@@ -1557,7 +1431,7 @@ pub mod pallet {
         #[pallet::weight(T::WeightInfo::force())]
         pub fn force(origin: OriginFor<T>, forcing_type: ForcingType) -> DispatchResult {
             Self::ensure_pallet_enabled()?;
-            T::ManagerOrigin::ensure_origin(origin)?;
+            ensure_root(origin)?;
```

Comment:
Will we still have the possibility to dispatch root calls on live env ? Will `Manager` still be relevant ?
Also please update the `fn` description

Reply:
Yes, via democracy pallet.

And of course, `sudo` 🙂

Reply:
Yes, via democracy pallet.

And of course, sudo 🙂